packaging/userspace/runpath_util.py

```diff
@@ -212,10 +212,13 @@ def _set_rpath_lief(
 				ent_runpath = lib.get(lief.ELF.DYNAMIC_TAGS.RUNPATH)
 				ent_runpath.paths = [runpath]
 			elif lib.has(lief.ELF.DYNAMIC_TAGS.RPATH):
-				# Changing the tag here causes a segfault when writing
 				ent_runpath = lib.get(lief.ELF.DYNAMIC_TAGS.RPATH)
 				ent_runpath.paths = [runpath]
-				runpath_is_rpath = True
+				if self.lief_info.fragile_builder:
+					# Changing the tag here causes a segfault when writing
```

Comment:
Is this something to worry about? How does one get into this situation?

Reply:
No, the if block here works around it for problematic versions of lief